data/json/items/gun/762.json

```diff
@@ -30,6 +30,38 @@
     "flags": ["MODE_BURST", "NEVER_JAMS"]
   },
   {
+    "id": "arx160",
+    "type": "GUN",
+    "symbol": "(",
```

Comment:
Field ordering is inconsistent with everything else under `gun`. This is required to make for easier item comparison and also to reduce potential for merge conflicts when multiple authors work on the same files.